scrapy/core/scraper.py

```diff
@@ -3,12 +3,14 @@
 
 import logging
 from collections import deque
+from collections.abc import Iterable
```

Comment:
What's your motivation for importing it from collections.abc, and not from typing?

Reply:
Just a habit, and it doesn't seem to make a difference unless `Iterable[foo]` is needed but I can change this to use `typing`.